lib/LoginStrategy.ts

```diff
@@ -266,8 +266,11 @@ export class LoginStrategy extends BaseStrategy {
         this.cacheServerClient?.address === userAddress
           ? []
           : await this.credentialResolver.eip191JwtsOf(userDid);
+      const claims = userClaims.filter((claim) =>
```

Comment:
I think the variable name can be improved here. `claims` is less descriptive than `userClaims`, so it unintuitive that is the result of a `filter`, IMO.
```suggestion
      const roleClaims = userClaims.filter((claim) =>
```